de.dlr.sc.virsat.model.edit/src/de/dlr/sc/virsat/model/dvlm/concepts/registry/ActiveConceptConfigurationElement.java

```diff
@@ -80,7 +80,17 @@ public String getXmi() {
 	}
 
 	/**
-	 *This method tells if the ACCE registers a given Concept. This method can be used
+	 * This method get the displayName of the concept
+	 * @return displayName of the concept
+	 */
+	public String getConceptName() {

```

Comment:
Even though it is only a very small method it may deserve a test case.
Could be a bit tricky since it is loading the concept from the plugin, but this method may be overriden for the test case.

Reply:
Agree.